### PR TITLE
p2 : nav , slider , some bug fixes

### PR DESCRIPTION
**Pull request by manojsarna**
p2 Components

Navigation Mobile Added
Slider Added

Some minor Bug Fixes

### CODE REVIEW

File: lib/Components/Navigation/navigation.css [hunk 1]
```diff
@@ -88,3 +88,135 @@
...
+}
+
+#sc-login{
+    white-space: nowrap;
+}
+
+#sc-logo {
+    white-space: nowrap;
+    display: inline-block;
+}
```
Comment (pankaj-prog) [inline lib/Components/Navigation/navigation.css:118]: do not use id for styling, make sure the specificity is as low as possible.

**Review by pankaj-prog — APPROVED** [review]
I like the way you used descriptive class names, good work

File: lib/Components/Navigation/navigation.css [hunk 2]
```diff
@@ -88,3 +88,135 @@
...
+
+.btn-social{
+    outline: none;
```
Comment (gauravsinhaweb) [inline lib/Components/Navigation/navigation.css:121]: give some outline value for accessibilty.

**Review by gauravsinhaweb — APPROVED** [review]
Readme looks clean and organised. just a minor change I have added. Good to merge!!

**Pull request merged**